ddemangle.d

```diff
@@ -58,10 +60,32 @@ const(char)[] demangleMatch(T)(Captures!(T) m)
     }
 }
 
-auto ddemangle(T)(T line)
-    if (is(T : const(char)[]))
+const(char)[] demangleMatchUnderscoreMissing(T)(Captures!(T) m)
+        if (is(T : const(char)[]))
```

Comment:
[DStyle](https://dlang.org/dstyle.html) would be to have indentation of constraints on the same level.

Reply:
I was just using `dfmt` ; i thought that was reflecting Dstyle?

Reply:
In theory, but `dfmt` has many bugs.

Reply:
ok fixed it, but IMO we should really prioritize making dfmt up to spec OR let dfmt define the spec, and ignore all debates about style so long dfmt was applied; modulo dfmt off/on in certain key places.